src/ewatercycle/_forcings/caravan.py

```diff
@@ -209,6 +209,16 @@ def generate(  # type: ignore[override]
 
         for temp in ["tas", "tasmin", "tasmax"]:
             ds_basin_time[temp].attrs.update({"height": "2m"})
+            #convert units to Kelvin for compatiabillity with NetCDF-CF conventions
```

Comment:
```suggestion
            #convert units to Kelvin for compatibility with CMOR MIP table units
```
The netCDF-CF convention just specifies you need to use units from [UDUNITS-2](https://docs.unidata.ucar.edu/udunits). degC is a valid unit there.

However, the units are standardized from the CMIP comparisons in so called "MIP tables". For example, the [CMIP6_day table](https://github.com/PCMDI/cmip6-cmor-tables/blob/main/Tables/CMIP6_day.json).